src/Simulations/simulation.jl

```diff
@@ -44,12 +45,16 @@ Keyword arguments
 
 - `wall_time_limit`: Stop the simulation if it's been running for longer than this many
                      seconds of wall clock time.
+- `minimum_relative_step`: time steps smaller than `Δt * minimum_relative_step` will be skipped.
+                           This avoids extremely high values when writing the pressure to disk.
+                           Default value is 0. See github.com/CliMA/Oceananigans.jl/issues/3593 for details.
 """
 function Simulation(model; Δt,
                     verbose = true,
                     stop_iteration = Inf,
                     stop_time = Inf,
-                    wall_time_limit = Inf)
+                    wall_time_limit = Inf,
+                    minimum_relative_step = 0)
```

Comment:
we should update the method signature in the docstring

Reply:
Good catch. Done!